### PR TITLE
Set up a 'Hello world' project with two apps

### PR DESCRIPTION
**Pull request by blase147**
In this PR, I;

- [Set up a 'Hello world' project with two apps](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md)
- Implemented a connection between [Ruby on Rails Back-end](https://github.com/blase147/hello-rails-back-end) and [React Front end](https://github.com/blase147/hello-react-front-end)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (addod19) [inline LICENSE:1]: - [ ] Kindly fix the MIT license link, it is currently broken

File: src/components/Greeting.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
```
Comment (addod19) [inline src/components/Greeting.jsx:2]: - [ ] Kindly make sure you install the following `react-redux`, `react-router-dom`, `@reduxjs/toolkit` before using it, it is displaying errors on the page. Check the package.json file to see all installed packages

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @blase147, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Correct Git-flow followed :100: 
- [ ] App component present with routes :1st_place_medal: 
- [ ] App component has a route that leads to the greeting components :rocket: 
- [ ] Store, action and reducer present :fireworks: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by addod19** [comment]
### NOTE:
@blase147, it is advisiable to make sure both PR's are approved before merging. We review both PR's. I can see you have merge the backend

**Comment by blase147** [comment]
I'm sorry about that. I didn't know about the double approval

File: LICENSE (hunk 1 above)
Comment (blase147) [reply LICENSE:1]: Thanks. The MIT license link is actually working...I double-checked it

File: src/components/Greeting.jsx (hunk 2 above)
Comment (blase147) [reply src/components/Greeting.jsx:2]: Thanks

**Review by Reem-lab — APPROVED** [review]
# Hi @blase147, 🤩🔥 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ] Great job working on the functionality 🔥🤩
- [ ] ALL the messages are displayed 🤩
- [ ] Following Gitflow✅
- [ ] Professional README 🔥

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**